### PR TITLE
Shortcut

### PR DESCRIPTION
**Pull request by SimranLahrani842**
Who will write entire word if a single , operator can perform the same function.
Useful when you want to join more than two/three joins.

### CODE REVIEW

File: c.md [hunk 1]
```diff
@@ -471,4 +471,9 @@ All functions defined under `<stdlib.h>` header file. These randomized functions
 
 * `gcc prog.c` : compiles prog.c into a.out, run result with  `./a.out`.
 * `gcc -o prog prog.c` : compiles prog.c into prog, run result with `./prog`.
-* `gcc -g -o prog prog.c` : as above, but allows for debugging.
\ No newline at end of file
+* `gcc -g -o prog prog.c` : as above, but allows for debugging.
+
+##use of fflush()
```
Comment (onecompiler-ops) [inline c.md:476]: Add space after `##` to fix markdown formatting

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: c.md (hunk 1 above)
Comment (SimranLahrani842) [reply c.md]: Added space after ##.Can you check now..

**Review by onecompiler-ops — APPROVED** [review]

**Comment by SimranLahrani842** [comment]
Thanks for your quick response...but please can you add labels of
hacktober2022 and other labels so that I can get quick response?...

**Comment by onecompiler-ops** [comment]
@SimranLahrani842 Added the lables

**Pull request merged**